src/components/policy/policy_external/src/sql_pt_representation.cc

```diff
@@ -1264,46 +1264,48 @@ bool SQLPTRepresentation::SaveConsumerFriendlyMessages(
   // According CRS-2419  If there is no “consumer_friendly_messages” key,
   // the current local consumer_friendly_messages section shall be maintained in
   // the policy table. So it won't be changed/updated
-  if (messages.messages.is_initialized()) {
-    utils::dbms::SQLQuery query(db());
-    if (!messages.messages->empty()) {
-      if (!query.Exec(sql_pt::kDeleteMessageString)) {
-        LOG4CXX_WARN(logger_, "Incorrect delete from message.");
-        return false;
-      }
-    }
+  if (!messages.messages.is_initialized()) {
+    LOG4CXX_INFO(logger_, "Messages list is empty");
```

Comment:
@AByzhynar fixed in https://github.com/smartdevicelink/sdl_core/pull/1615/commits/36a37e964f03b3c01decee10c80a67b8c5ec1590